yang/frr-zebra.yang

```diff
@@ -511,6 +573,36 @@ module frr-zebra {
     }
   }
 
+  grouping ribs {
+    container ribs {
+      description
+        "RIBs supported by FRR.";
+      list rib {
+        key "table-id";
+        leaf table-id {
+          type uint32;
+          description
+            "Routing Table id.";
+        }
+
+        leaf afi-safi-name {
+          type identityref {
+            base afi-safi-type;
+          }
+          mandatory true;
+          description
+            "AFI, SAFI name.";
+        }
```

Comment:
Should afi-safi-name be key as well? How can we get routes from only for a family?

Reply:
If this is not key then should it not be config false?

Reply:
Added af-safi-name as a key to list of routes.

Reply:
I don't think it should be called name if it's really an identity and not e.g. a string, but we can come back to this at a later date.